examples/sycl/05_pvc_gemm_with_epilogues/05_pvc_gemm_with_epilogue_lincombdeeltact.cpp

```diff
@@ -363,90 +459,17 @@ int main(int argc, const char** argv)
-
-  using GmemTiledCopyA = XE_2D_U16x32x32_LD_N;
-  using GmemTiledCopyB = XE_2D_U16x32x32_LD_V;
-
-  // Workgroup-level tile
-  using TileShape = Shape<_256, _256, _32>;
-
-  using TiledMma =
-      typename TiledMMAHelper<MMA_Atom<XE_8x16x16_F32BF16BF16F32_TT>, Layout<TileShape>,
-                                    Layout<Shape<_8, _4, _1>, Stride<_4, _1, _0>>>::TiledMMA;
-
-  constexpr int PipelineStages = 2;
-  using GEMMDispatchPolicy = cutlass::gemm::MainloopIntelPVC<PipelineStages>;
-  using EpilogueDispatchPolicy = cutlass::epilogue::IntelPVCEpilogue;
-
-  using CopyOpG2R = XE_2D_U32x8x16_LD_N; 
-  using EpilogueOp = cutlass::epilogue::fusion::LinCombDeEltAct<
-      LayoutC,
-      sum_vals,
-      ElementOutput,
-      ElementComputeEpilogue>;
-  static_assert(std::is_same_v<EpilogueOp::ElementOutput, ElementOutput>);
-
-  using EpilogueTile = decltype(take<0,2>(TileShape{}));
-
-  using FusionCallBacks = cutlass::epilogue::fusion::FusionCallbacks<
-          EpilogueDispatchPolicy,
-          EpilogueOp,
-          TileShape,
-          EpilogueTile, 
-          CopyOpG2R
-          >;
-  using CollectiveEpilogue = cutlass::epilogue::collective::CollectiveEpilogue<
-          EpilogueDispatchPolicy,                 // IntelPVCEpilogue
-          TileShape,                              // CtaTileMNK
-          ElementAccumulator,                     // ElementC
-          cutlass::gemm::TagToStrideC_t<LayoutC>, // StrideC
-          ElementOutput,                          // ElementD
-          cutlass::gemm::TagToStrideC_t<LayoutD>, // StrideD
-          FusionCallBacks,                        // FusionCallBacks
-          CopyOpG2R,                              // CopyOpG2R
-          void,                                   // SmemLayoutAtomC
-          void,                                   // CopyOpS2R
-          XE_2D_U32x8x16_ST_N,                    // CopyOpR2G
-          void,                                   // SmemLayoutAtomD
-          void>;                                  // CopyOpR2S
-
-// Mainloop
-  using CollectiveMainloop = cutlass::gemm::collective::CollectiveMma<
-          GEMMDispatchPolicy,
-          TileShape,
-          ElementInputA,
-          cutlass::gemm::TagToStrideA_t<LayoutA>,
-          ElementInputB,
-          cutlass::gemm::TagToStrideB_t<LayoutB>,
-          TiledMma,
-          GmemTiledCopyA, void, void, cute::identity,  // A
-          GmemTiledCopyB, void, void, cute::identity   // B
-  >;
-
-  using GemmKernel = cutlass::gemm::kernel::GemmUniversal<
-  Shape<int, int, int, int>,
-  CollectiveMainloop,
-  CollectiveEpilogue
-  >;
-
-  using Gemm = cutlass::gemm::device::GemmUniversalAdapter<GemmKernel>;
 
-  ExampleRunner<Gemm> runner;
-
-  CUTLASS_CHECK(runner.run(options, hw_info));
+  if (options.activation == Activation::Sum){
+    using SumGemm = cutlass::gemm::device::GemmUniversalAdapter<GemmKernel<sum_vals>>;
+    ExampleRunner<SumGemm, sum_vals> runner;
```

Comment:
`sum_vals` can be replaced with `cutlass::plus`. Likewise `diff_vals` could be replaced with `cutlass::minus` (or removed since its unused).

Reply:
I'd like to keep sum_vals as an example

Reply:
Sure :+1: